### PR TITLE
Feature/trello integration

### PR DESCRIPTION
**Pull request by JDHProjects**
PR for Aptem module 2

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+TRELLO_TOKEN=trello-token
+TRELLO_API_KEY=trello-api-key
+BOARD_ID=trello-board-id
+COMPLETED_LIST=trello-completed-list
+NOT_STARTED_LIST=trello-not-started-list
```
Comment (JackMeadDev) [inline .env.template]: Watch out that you've changed the names of these last three variables in your code to be prefixed by `TRELLO_` so this is slightly misleading!

File: README.md [hunk 2]
```diff
@@ -26,14 +26,22 @@ The project uses a virtual environment to isolate package dependencies. To creat
...
 ```
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+Some additional variables are required to be able to integrate with Trello:
+* `TRELLO_TOKEN` - Trello token
+* `TRELLO_API_KEY` - Trello API key
+* `TRELLO_BOARD_ID` - Board to hold both below lists
+* `TRELLO_COMPLETED_LIST` - List to hold completed tasks
+* `TRELLO_NOT_STARTED_LIST` - List to hold not yet started tasks
```
Comment (JackMeadDev) [inline README.md:43]: Thanks for the clear instructions 😄

File: todo_app/data/trello_items.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import requests
+import os
+
+TRELLO_BASE = "https://api.trello.com/1"
+BOARD_ID = "63dd25f4eaaf087231fa1c44"
```
Comment (JackMeadDev) [inline todo_app/data/trello_items.py]: It looks to me like this is never used, can we get rid of it in favour of the run-time loaded environment variables?

File: todo_app/templates/index.html [hunk 4]
```diff
@@ -7,21 +7,51 @@ <h1 class="display-4">To-Do App</h1>
...
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
+  <div class="row" style="display: flex;text-align: center;">
+    <div style="width: 45%;">
+      <h2>Not Started</h2>
         {% for task in tasks %}
-          <li>{{ task.title }}</li>
+          {% if task.status == "Not Started" %}
```
Comment (JackMeadDev) [inline todo_app/templates/index.html]: Really minor but here you have a couple of examples of "magic strings" that are easy to typo (e.g. incorrect capitalisation) or forget to change later - would switching this for a property/method on the Item class called "is_not_started" be neater?

**Review by JackMeadDev — APPROVED** [review]
Nice work, I was easily able to run your app following the instructions and view/manage my to-dos in Trello using it 😄 I've left a couple of optional thoughts below, but happy for you to merge when you're ready

File: todo_app/templates/index.html (hunk 4 above)
Comment (JDHProjects) [reply todo_app/templates/index.html]: Of course! No idea why I didn't do that in the first place

File: .env.template (hunk 1 above)
Comment (JDHProjects) [reply .env.template]: Apologies, updated

**Comment by JDHProjects** [comment]
Thanks for the feedback @JackMead , I've actioned your suggestions

**Pull request merged**